index.css

```diff
@@ -4,8 +4,70 @@ body {
   font-size: 100%;
 }
 
+main {
```

Comment:
- [ ][OPTIONAL] Good job team 👍👏. It would be better if you add a screenshot to the readme file to improve it 🍀